packages/plugins/example/src/index.ts

```diff
@@ -6,18 +6,14 @@ registerPlugin({
     SNSAdaptor: {
         load: () => import('./SNSAdaptor'),
         hotModuleReload: (hot) =>
-            import.meta.webpackHot &&
-            import.meta.webpackHot.accept('./SNSAdaptor/index', () => hot(import('./SNSAdaptor'))),
+            import.meta.webpackHot?.accept('./SNSAdaptor/index', () => hot(import('./SNSAdaptor'))),
```

Comment:
This code will be compiled from

```js
import.meta.webpackHot?.accept()
```

To 

```js
var _a;
(_a = import.meta.webpackHot) === null || _a === void 0 ? void 0 : _a.accept();
```

And it might let dead code elimination fail, I didn't verify that. Can you verify that before the change?

Reply:
I used the below code and ran it to build locally. It looks like both methods were eliminated.

```typescript
alert(String(import.meta.webpackHot))

import.meta.webpackHot && (() => alert('METHOD 1'))()

const a = import.meta.webpackHot
a === null || a === undefined ? undefined : alert('METHOD 2')
```

Reply:
OK